src/xdocs/config_imports.xml

```diff
@@ -847,6 +847,16 @@ public class InputIllegalImport { }
         </p>
 
         <p>
+          Short description of the behaviour:
+          <ul>
+              <li>Check starts checking from the longest matching subpackage described inside import control file to package defined in class file (later 'current subpackage').</li>
```

Comment:
"(later 'current subpackage')" is misplaced.
"longest matching subpackage" ==> "longest matching subpackage (later 'current subpackage')"